src/Sylius/Component/Grid/Data/ExpressionBuilderInterface.php

```diff
@@ -54,6 +54,22 @@ public function equals(string $field, $value);
      */
     public function notEquals(string $field, $value);
 
+    /**
+     * @param string $field
+     * @param string $secondField
+     *
+     * @return mixed
+     */
+    public function fieldsEquals(string $field, string $secondField);
```

Comment:
Adding a method to an interface is a BC break unfortunately :/

Reply:
Oh, indeed... What would be the best to do it then ?